<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSettings</class>
 <widget class="QDialog" name="VideoSettings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Settings</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>DMG Color Palette</string>
   </property>
   <widget class="QPushButton" name="pushButtonColor1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>32</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonColor2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>32</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonColor3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>32</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonColor4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>32</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxFilter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilinear Filtering</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMix">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mix Frames</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VideoSettings</receiver>
   <slot>PressedOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VideoSettings</receiver>
   <slot>PressedCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColor1</sender>
   <signal>clicked()</signal>
   <receiver>VideoSettings</receiver>
   <slot>Color1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColor2</sender>
   <signal>clicked()</signal>
   <receiver>VideoSettings</receiver>
   <slot>Color2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColor3</sender>
   <signal>clicked()</signal>
   <receiver>VideoSettings</receiver>
   <slot>Color3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColor4</sender>
   <signal>clicked()</signal>
   <receiver>VideoSettings</receiver>
   <slot>Color4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PressedOK()</slot>
  <slot>PressedCancel()</slot>
  <slot>Color1()</slot>
  <slot>Color2()</slot>
  <slot>Color3()</slot>
  <slot>Color4()</slot>
 </slots>
</ui>
